flink-cdc-common/src/main/java/com/ververica/cdc/common/utils/SchemaUtils.java

```diff
@@ -32,6 +33,7 @@
 import java.util.stream.Collectors;
 
 /** Utils for {@link Schema} to perform the ability of evolution. */
+@PublicEvolving
```

Comment:
As we are promoting this utility to public, I think we need to be careful about the API, so I have concerns about the method `applySchemaChangeEvent` with parameter `ignoreException`. I prefer to remove this one, as I can't see any use cases for ignoring exceptions. Currently there's no usage of it, and the warning log is empty if the exception is ignored, which confuses me a bit. 

A better behavior would be clearly defining the exception on the method (`throws XXXException`) and letting the caller catch and handle it.